lib/base/error-handler.js

```diff
@@ -4,122 +4,241 @@
-        "INVALID_OBJECT" : "Object format error",
-        "INVALID_MODE" : "Please specify an option, either '--add' or '--modify'",
-        "INVALID_CAPTURE_FORMAT" : "Please specify file extension, either 'png' 'bmp' or 'jpg'",
-
-        // set value & type error
-        "EMPTY_PROFILE" : "Profile is empty",
-        "EMPTY_VALUE" : "Please specify a value",
-        "EXISTING_VALUE" : "The specified value already exists",
-        "EXISTING_FILETYPE_PATH" : "A file with the same name at the specified location already exists",
-        "NOT_DIRTYPE_PATH" : "The speficied path is not a directory",
-        "NOT_EXIST_PATH" : "The specified path does not exist",
-        "NOT_EXIST_SSHKEY_PASSWD" : "Private key file or password does not exist",
-        "NOT_EXIST_DISPLAY" : "No existing displayId from getSessionList",
-        "NOT_SUPPORT_ENYO" : "Enyo app packaging is not supported",
-        "NOT_SUPPORT_AUTHTYPE" : "Not supported auth type",
-        "NOT_SUPPORT_RUNNINGLIST" : "Not supported method to get running app information",
-        "NOT_USE_WITH_OPTIONS" : "Do not use together with options",
-
-        "USE_WITH_OPTIONS" : "Use together with options",
-        "USE_PKGID_PKGINFO" : "packageId must be provided by using either the '--pkgid' or the '--pkginfofile' option",
-        "USE_GDB" : "Please use GDB to debug a native serivce",
-        "NOT_IMPLEMENTED" : "Not implemented",
-        "NOT_HANDLE_FILE" : "Don't know how to handle",
-        "NOT_OVERWRITE_DIR" : "Cannot overwrite the directory",
-
-        "UNMATCHED_DEVICE" : "No matched device",
-        "UNMATCHED_DISPLAY_AFFINITY" : "Please use the same value for \"display\" and \"displayAffinity\"",
-
-        "FAILED_CALL_LUNA" : "luna-send command failed",
-        "FAILED_GET_PORT" : "Failed to get Debug port",
-        "FAILED_GET_SVCPATH" : "Failed to get service installation path",
-        "FAILED_GET_SSHKEY" : "Failed to get ssh private key",
-        "FAILED_TRANSMIT_FILE" : "File transmission error. Please try again",
-        "FAILED_CREATE_DIR" : "Cannot create directory in the destination path - permission denied",
-        "FAILED_REMOVE_PACKAGE" : "No packages installed or removed",
-        "FAILED_FIND_SERVICE" : "Failed to find a service with the specified name",
-
-        "CONNECTED_MULTI_DEVICE" : "Multiple devices are connected by novacom. Please specify the target name",
-        "SET_DEFAULT_MULTI_DEVICE" : "Multiple devices are set to default target. Please reset device list",
-        "NEED_ROOT_PERMISSION" : "Unable to connect to the target device. root access required",
-        "MISSING_CALLBACK" : "Missing completion callback",
-        "USING_WEBINSPECTOR" : "Web Inspector is already connected with another browser. Please close the previous connection",
-        "UNKNOWN_OPERATOR" : "Unknown operator",
-        "UNABLE_USE_SFTP" : "Unable to use sftp",
+            // service's error message, common
```

Comment:
How about managing ErrMsgMap as a separate file like command-service.json?

Reply:
@ssuminahn , Let me think about this change.

Reply:
I have a talk with @ssuminahn , I'll keep this map in error-handler.js